app/presenters/rejection_reasons/rejection_reasons_presenter.rb

```diff
@@ -77,6 +78,8 @@ def nested_advice_reasons(reason)
     end
 
     def valid_tailored_advice_reason_id(reason_id)
+      return if reason_id.in? NO_TAILORED_ADVICE_CODES
```

Comment:
Could you please add a spec for this to `rejection_reasons_presenter_spec`, and the `candidate_mailer_application_rejected_tailored_advice_spec`? This is a very complicated and easily breakable class, so it is important to have everything tested.